ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/correction/RemoveBuildEntryResolution.java

```diff
@@ -57,4 +59,27 @@ protected void createChange(Build build) {
 		}
 	}
 
+	@Override
+	public void run(IMarker marker) {
+		super.run(marker);
+		this.marker = marker;
+	}
+
+	@Override
+	public IMarker[] findOtherMarkers(IMarker[] markers) {
+		HashSet<IMarker> mset = new HashSet<>(markers.length);
```

Comment:
What do you think about using a Stream-pipeline in this method?
I think this would be more compact and expressive.